opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -31,35 +31,64 @@
 import os
 import sys
 import opentimelineio as otio
+from collections import Iterable
 
 lib_path = os.environ.get("OTIO_AAF_PYTHON_LIB")
 if lib_path and lib_path not in sys.path:
     sys.path += [lib_path]
```

Comment:
As noted by @shahbazk8194 earlier today, this should be a prepend instead of an append.